README.md

```diff
@@ -16,18 +16,44 @@ Blog posts are located in [`./src/pages/blog`](https://github.com/ParalelnaPolis
 
 ## Development
 
+### Prerequisities
+
+- you need [node with npm (node package manager)](https://nodejs.org/en/download/) installed to be able to build and run website locally on you computer. Let's stick with LTS (Long Term Support) version.
+- [git](https://git-scm.com/downloads) is needed to commit new changes; you can find also some [GUI clients](https://git-scm.com/downloads/guis) that may help you with that.
+  - how to work with git? [the simple guide - no deep shit!](http://rogerdudler.github.io/git-guide/)
+- your favourite text editor (humbly recommending [vscode](https://code.visualstudio.com/))
+
+### Installation
+
+In the terminal run:
```

Comment:
You sure it's Node.js terminal? I don't think it exists.

OK, probably better name is _command line_.

On Mac/Linux it's Terminal that runs command line. On Windows it should work in Command Prompt, if not than try Git terminal.

I don't currently have Windows, so I cannot try. If you do have, please try and let me know which one works.

Reply:
whatever this is... :)  
![image](https://user-images.githubusercontent.com/24230613/43771507-c4251114-9a3f-11e8-8ca7-bb6d82788889.png)

ok, tried Git terminal: 

Lukas@Lukas-IdeaPC MINGW64 ~
$ git clone git@github.com:ParalelnaPolisKE/web.git
Cloning into 'web'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

Reply:
Yep, you need to add your public key to your github account to be able to clone. (This will be new step in installation probably).

The process of generating key is specific for every OS, so it's hard to lead you now.

Reply:
BTW. I haven't known Node.js terminal exists, wow 😃

Reply:
ok, I suggest to add: https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/ to this readme step for noobs like me.